easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
@boegel I've made a couple of small changes in the easyblock and easyconfig, so now you can specify a list of host compilers, and the easyblock will create the appropriate wrappers based on that.